src/index.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable quotes */
+import "./css/style.css";
+import renderMovies from "./modules/pageBody.js";
+import likeMovie from "./modules/likeFeature.js";
+/* eslint-disable linebreak-style */
+/* eslint-disable max-len */
+/* eslint-disable linebreak-style */
+/* eslint-disable no-multiple-empty-lines */
+/* eslint-disable linebreak-style */
+/* eslint-disable array-callback-return */
+/* eslint-disable arrow-body-style */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable quotes */
```

Comment:
- [ ] Please remove `eslint-disable` lines as that defies the purpose of using linters in the first place. (This goes to all files that `eslint-disable` is used in)

Reply:
Okey Thanks